app/main.py

```diff
@@ -1,3 +1,38 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        data = json.load(file)
+    fuel_price = data["FUEL_PRICE"]
+    customers = [
+        Customer(
+            customer["name"],
```

Comment:
It's better practice to use keyword argumets when dataclasses are initialized. Using them, you are not needed to remember an order in which arguments should be passed